main_routers/workshop_router.py

```
@@ -0,0 +1,1134 @@
+                                if hasattr(result, 'description') and result.description:
+                                    item_info['description'] = result.description.decode('utf-8', errors='replace')
+                                if hasattr(result, 'timeCreated'):
+                                    item_info['timeAdded'] = int(result.timeCreated)
+                                if hasattr(result, 'timeUpdated'):
+                                    item_info['timeUpdated'] = int(result.timeUpdated)
+                                if hasattr(result, 'steamIDOwner'):
+                                    item_info['steamIDOwner'] = str(result.steamIDOwner)
+                                if hasattr(result, 'fileSize'):
+                                    item_info['fileSizeOnDisk'] = int(result.fileSize)
+                                logger.info(f"成功获取物品 {item_id} 的详情信息")
+                        except Exception as query_error:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)